tmt/steps/prepare/install.py

```diff
@@ -165,17 +165,25 @@ def go(self, guest):
         packages = self.get('package', [])
         repo_packages = []
         local_packages = []
+        repo_debuginfos = []
```

Comment:
What about calling it `debuginfo_packages` to make the content more obvious?

Reply:
Done. Thanks